app/scripts/modules/core/src/search/infrastructure/RecentlyViewedItems.tsx

```diff
@@ -86,7 +86,7 @@ export class RecentlyViewedItems extends React.Component<IRecentlyViewedItemsPro
   }
 
   private handleResultClick(categoryName: string): void {
-    ReactGA.event({ category: 'Global Search', action: `Recent item selected from ${categoryName}` });
+    ReactGA.event({ category: 'Primary Search', action: `Recent item selected from ${categoryName}` });
```

Comment:
@christopherthielen are we using a specific category for other search related stuff that doesn't fall under global search? Just want to make sure we don't fragment categories.

Reply:
Honestly, I'm not sure.